src/Bpi/ApiBundle/Controller/ChannelController.php

```diff
@@ -481,7 +481,7 @@ public function addNodeToChannelAction()
         $dm->flush();
 
         $facetRepository = $this->getRepository('BpiApiBundle:Entity\Facet');
-        $facetRepository->addChannelToFacet($channel->getChannelName(), $params['nodes']);
+        $facetRepository->addChannelToFacet($channel->getId(), $params['nodes']);
```

Comment:
I suggest to change method arguments names.
https://github.com/inleadmedia/rest-api/blob/develop/src%2FBpi%2FApiBundle%2FDomain%2FRepository%2FFacetRepository.php#L298